server/controllers/user.controllers.js

```diff
@@ -204,6 +207,38 @@ const UserController = {
       })
       .catch((error) => next(error));
   },
+
+  verifyVendorStep(req, res, next) {
+    const verificationInfo = req.locals;
+    const adminId = req.user._id;
+    verifyVendorStep({ ...verificationInfo, adminId })
+      .then((vendor) => {
+        res
+          .status(httpStatus.OK)
+          .json({ success: true, message: 'Company information verified', vendor });
```

Comment:
We are not sure if the vendor is  a company or an individual
```suggestion
          .json({ success: true, message: 'Vendor information verified', vendor });
```